begrun.c

```diff
@@ -213,13 +217,16 @@ set_units(void)
     message(0, "UnitDensity_in_cgs = %g \n", All.UnitDensity_in_cgs);
     message(0, "UnitEnergy_in_cgs = %g \n", All.UnitEnergy_in_cgs);
     message(0, "Photon density OmegaG = %g\n",All.CP.OmegaG);
-    message(0, "Massless Neutrino density OmegaNu0 = %g\n",All.CP.OmegaNu0);
+    if(!All.MassiveNuLinRespOn)
+        message(0, "Massless Neutrino density OmegaNu0 = %g\n",get_omega_nu(&All.CP.ONu, 1));
     message(0, "Curvature density OmegaK = %g\n",All.CP.OmegaK);
     if(All.CP.RadiationOn) {
         /* note that this value is inaccurate if there is massive neutrino. */
+        double OmegaTot = All.CP.OmegaG + All.CP.OmegaK + All.CP.Omega0 + All.CP.OmegaLambda;
+        if(All.CP.MNu[0] + All.CP.MNu[1] + All.CP.MNu[2] == 0)
+            OmegaTot += get_omega_nu(&All.CP.ONu, 1);
```

Comment:
ONu and MNu are not independent parameters -- Onu depends on the number of massless species in MNu. From this perspective this line here is confusing -- what if I only have 1 massless species?

Reply:
Yes - this is a wrong condition. It should read 
if(!All.MassiveNuLinRespOn)
OmegaTot += get_omega_nu()
Apologies, I wrote this late last night.